doc/source/user_guide/pool.rst

```diff
@@ -30,6 +30,17 @@ at the current directory within their own isolated directories:
     >>> pool = MapdlPool(10, nproc=1, run_location=my_path)
     Creating Pool: 100%|########| 10/10 [00:01<00:00,  1.43it/s]
 
+Additionally, you can group already running MAPDL instances into an
+:class:`MapdlPool <ansys.mapdl.core.pool.MapdlPool>` by specifying
```

Comment:
```suggestion
:class:`MapdlPool <ansys.mapdl.core.pool.MapdlPool>` instance by specifying
```
A noun should follow a code entity so that in the rendered doc, the text is easier to read and understand.